src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -0,0 +1,328 @@
+	            help="static interface subcommands"
+	            ptype="SUBCOMMAND"
+	            mode="switch"	    
+		    >
+                    <PARAM
+                        name="Ethernet"
+                        help="static interface Ethernet id - ethernet"
+		        ptype="SUBCOMMAND"
+		        mode="subcommand"
+		        >
+		        <PARAM
+		            name="phy-if-id"
+                            help="Static interface ethernet id-phy-if-id"
+			    ptype="UINT"
+			    >
+	                 </PARAM>
+		    </PARAM>
+                    <PARAM
+                        name="Port-channel"
+                        help="static port-channel id - pc"
+		        ptype="SUBCOMMAND"
+		        mode="subcommand"
+		        >
+		        <PARAM
+		            name="po-id"
+                            help="Static interface port-channel id po-id"
+			    ptype="UINT"
+			    >
+		        </PARAM>
+	            </PARAM>
+		</PARAM>
+            </PARAM>
+        </PARAM>
+        <ACTION>
+		if test "${s-subcommands}" = "Address"; then&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static address ${mac-addr}&#xA; 
+		elif test "${s-subcommands}" = "VLAN"; then&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static vlan ${vlan-id}&#xA; 
+                elif test "${s-subcommands}" = "Interface"; then&#xA;
+                    if test "${interface-subcommand}" = "Port-channel"; then&#xA;
+                        python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static interface Port-channel${po-id}&#xA;
+                    else&#xA;
+                        python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static interface Ethernet${phy-if-id}&#xA;
+                    fi&#xA;
+                else&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 static ${__full_line}&#xA;
+	        fi	
+        </ACTION>
+    </COMMAND>
+
+<!--====================show mac address-table dynamic commands ======================-->
+    <COMMAND
+        help="mac address-table for dynamic commands"
+        name="show mac address-table dynamic">
+        <PARAM
+            name="d-subcommands"
+            help="dynamic subcommands"
+            mode="switch"
+	    ptype="SUBCOMMAND"
+	    optional="true"
+            >
+            <PARAM
+                name="Address"
+                help="Address of dynamic mac"
+                ptype="SUBCOMMAND"
+                mode="subcommand"
+                >
+                <PARAM
+                    name="mac-addr"
+                    help="dynamic Mac address"
+                    ptype="MAC_ADDR"
+                    >
+                </PARAM>
+            </PARAM>
+            <PARAM
+                name="VLAN"
+                help="VLAN ID (1-128)"
```

Comment:
Please correct the VLAN ID range.

Reply:
Addressed this issue in the latest commit.